ckan/tests/factories.py

```diff
@@ -299,6 +298,9 @@ def _create(cls, target_class, *args, **kwargs):
 
         context = {'user': _get_action_user_name(kwargs)}
 
+        if 'type' not in kwargs:
+            kwargs['type'] = 'organization'
```

Comment:
There's [`dict.setdefault`](https://docs.python.org/2/library/stdtypes.html#dict.setdefault) for this pattern.